salt/preflight/defaults.yaml

```diff
@@ -0,0 +1,32 @@
+destructive: False      # define if /var/lib/etcd if should be removed
+upgrade: False          # define if we're on an update case
```

Comment:
At least this one (upgrade) should be part of the global defaults.yaml.

It's maybe make sense to have a separate defaults.yaml file for preflight or include it directly in the global one.
I personnaly don't know but if you include it in the global one, I suggest to have a top section `preflight` or something else.